calibrators/marker_radar_lidar_calibrator/README.md

```diff
@@ -0,0 +1,168 @@
+
+The images below illustrate the process of radar background model construction and radar foreground extraction that is described in Step 1 and Step 2.
+The blue 3d voxels, which are shown in a 2D grid in the images, are denoted as the background voxel if radar objects are in the voxels during the background model construction. Once the background model is constructed, it becomes straightforward to extract the foreground points in the calibration area. For the lidar, the background model and foreground extraction process are the same as the radar process described above.
+
+<table>
+  <tr>
+    <td><img src="../docs/images/marker_radar_lidar_calibrator/background_construction.svg" alt="background_construction" width = 700px height = 300px ></td>
+    <td><img src="../docs/images/marker_radar_lidar_calibrator/foreground_extraction.svg" alt="foreground_extraction" width = 700px height = 300px ></td>
+   </tr>
+   <tr>
+    <td><p style="text-align: center;">Background model construction.</p></td>
+    <td><p style="text-align: center;">Foreground extraction</p></td>
+  </tr>
+</table>
+
+### Step 3: Matching and filtering
+
+Since it is not possible to directly differentiate reflector detections, whether there are multiple targets in the calibration area or if the detections are from humans or radar reflectors, we rely on the initial calibration to pair each lidar detection with its closest radar detection, and vice versa. A detection pair is accepted if they are mutually their closest matches. Once a match is made, it is evaluated against existing hypotheses (monitored by a Kalman filter): if it aligns with an existing hypothesis, that hypothesis is updated; if it does not align with any, a new hypothesis is created. When a hypothesis achieves convergence, it is added to the calibration list.
+
+### Step 4: Calibration
+
+After matching detection pairs from the sensors, we can compute the transformation between them using rigid transformation estimation algorithms. Currently, we support two methods: a 2d SVD-based approach and a yaw-only rotation approach.
+
+For the 2d SVD-based method, since radar detections lack a z component, we reduce the problem to 2d by setting the z component of lidar detections to zero. We then estimate the rigid transformation using the SVD-based method provided by PCL, which leverages SVD to find the optimal rotation component and then calculates the translation component based on the rotation.
+
+The yaw-only rotation method, on the other hand, calculates the average yaw angle difference of all pairs and estimates the transformation, considering only yaw rotation, between the sensors. Generally, the 2d SVD-based method is preferred when valid; otherwise, the yaw-only rotation method is used as the calibration output.
```

Comment:
The description is wrong. As it is written it probably will be interpreted as somethings different from what we do 
When you talk about angles, rotation, etc, you need to explain to which one do you refer.

Reply:
considering only yaw rotation, between the sensors
Actually this is not true right? Is that not the parallel frame?

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/6c8622592bf9b43d2b256f743c9ec2a12f3abfed and https://github.com/tier4/CalibrationTools/pull/164/commits/99da6e1071c2155a3c8cc8ca67d97b270e576816

Reply:
Besides there being grammatical errors, the parallel frame was not explained. You need to describe the motivation, design, and potential choices. It should also be `quoted` so that the user reads as a term defined here rather than a literal meaning

Reply:
Thanks, I rewrite the whole paragraph in https://github.com/tier4/CalibrationTools/pull/164/commits/4f7a533dae98254e0ffcc87c385fe38e25d5fd4e.